vaadin-text-field-flow/src/test/java/com/vaadin/flow/component/textfield/tests/NumberFieldTest.java

```diff
@@ -97,4 +97,21 @@ public void assertMaxValidation() {
         field.setValue(99.999);
         Assert.assertFalse(field.isInvalid());
     }
+
+    @Test
+    public void testPropertyValue() {
```

Comment:
This test name is not very descriptive for me. It should mention "formatting" since that is the functionality it is testing.

I suggest `setValue_valuePropertyFormatted`, or `testValueFormatting` or something like that.

Reply:
Done